src/services/codefixes/importFixes.ts

```diff
@@ -435,6 +437,23 @@ namespace ts.codefix {
                     return position;
                 }
 
+                function getSingleQuoteStyleFromExistingImports() {
+                    const firstModuleSpecifier = forEach(sourceFile.statements, node => {
+                        switch (node.kind) {
+                            case SyntaxKind.ImportDeclaration:
+                                return (<ImportDeclaration>node).moduleSpecifier;
+                            case SyntaxKind.ImportEqualsDeclaration:
+                                const moduleReference = (<ImportEqualsDeclaration>node).moduleReference;
+                                return moduleReference.kind === SyntaxKind.ExternalModuleReference ? moduleReference.expression : undefined;
+                            case SyntaxKind.ExportDeclaration:
+                                return (<ExportDeclaration>node).moduleSpecifier;
+                        }
+                    });
+                    if (firstModuleSpecifier && isStringLiteral(firstModuleSpecifier)) {

```

Comment:
Hmm. I did this check because we might parse an invalid module specifier but we treat it as a grammar error rather than a parse error.